### DISCUSSION TITLE
TreeView控件，如何控制某些叶子节点不显示复选框

### DISCUSSION THREAD

**Discussion by shihuixian** [BlazorComponent/MASA.Blazor#1499] [Q&A] [answered]
现在设置了Selectable之后，所有的节点都会显示一个复选框，有没有办法控制某些节点不显示复选框？

**Comment by capdiem** [comment]
@shihuixian 用 @bind-Active, Activatable 和 MultipleActive

**Reply by shihuixian** [reply]
可能我描述的不够清楚，我想要实现的效果是这样，红框里的复选框不希望显示
![不显示通道复选框](https://github.com/masastack/MASA.Blazor/assets/40941172/6990b3c1-782e-4906-997e-8fa274e18209)
这几个字段我试了一下，没有办法隐藏指定节点的复选框。例子代码如下：
![image](https://github.com/masastack/MASA.Blazor/assets/40941172/cfcbca3f-2a1f-44c4-97bb-b76f71a519fe)

**Comment by capdiem** [comment, marked as answer]
@shihuixian selectable开启时没有办法关闭，你可以尝试下不开启selectabe的时候，使用LabelContent或者PrependContent插槽里使用SimpleCheckbox，得自己控制激活项~

**Reply by shihuixian** [reply]
谢谢，通过PrependContent插槽解决了这个问题。这个PrependContent的宽度似乎是有限制的，在加了复选框后，如果还要加一个图标就显得比较窄了。
